src/lib/ngsi/MaxFailsLimit.cpp

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Are src/lib/ngsi/MaxFailsLimit.cpp and src/lib/ngsi/MaxFailsLimit.h really needed? It seems your are not actually using them.

Maybe they are a leftover of previous implementation and they are no longer needed? That would be desirable, in which case please remove them.

Reply:
Removed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/6b68a8c9ded45dd2e30c033f4d5000cddfab19de